eclipse-theia/theia-blueprint
# [Jenkins] Make CI more robust

#### What it does

A few small improvements that will, hopefully, help stabilize
the CI:
- added a few volumes so that some strategix folders are writable by the CI job
- added a retry mechanism for the `yarn` build call, so that it will automatically retry a couple of times in case of failure.

#### How to test


Check that CI has passed :) 

Existing PRs that have build issue might benefit from being re-based to include these improvements, once merged.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
